src/plugins/telemetry_management_section/public/components/telemetry_management_section.tsx

```diff
@@ -197,12 +201,16 @@ export class TelemetryManagementSection extends Component<Props, State> {
         />
       </p>
       <p>
-        <EuiLink onClick={this.toggleExample}>
-          <FormattedMessage
-            id="telemetry.seeExampleOfWhatWeCollectLinkText"
-            defaultMessage="See an example of what we collect"
-          />
-        </EuiLink>
+        <FormattedMessage
+          id="telemetry.seeExamplesOfWhatWeCollect"
+          defaultMessage="See examples of the {clusterData} and {endpointSecurityData} that we collect."
+          values={{
+            clusterData: <EuiLink onClick={this.toggleExample}>cluster data</EuiLink>,
+            endpointSecurityData: (
+              <EuiLink onClick={this.toggleSecurityExample}>endpoint security data</EuiLink>
+            ),
```

Comment:
Should we add new translation IDs for `cluster data` and `endpoint security data` as well?

Reply:
Thanks! Updated in 0fae604